components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-authentication-flow.jsp

```diff
@@ -259,7 +259,7 @@
                             <a class="step_order_header" href="#">Step <%=step.getStepOrder()%>
                             </a>
                             <a href="#" class="delete_step icon-link"
-                               style="background-image: url(images/delete.gif);float:right;width: 9px;"></a>
+                               style="background-image: url(images/delete.gif);float:right;width: 9px;" data-step-no="<%=step.getStepOrder()%>"></a>
```

Comment:
Is this change related to this PR? There are some other files as well, Please check.